app/client/cypress/integration/about/about.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('About Page Tests', () => {
+  it('About Page Back Button Works', () => {
+    cy.visit('/');
+    cy.findByText('Let’s get started!').should('exist');
+    cy.url().should('equal', `${document.location.origin}/`);
+
+    cy.findByText('About').click();
+    cy.findByText('About How’s My Waterway').should('exist');
+    cy.url().should('equal', `${document.location.origin}/about`);
+
+    cy.findByText('Back').click();
```

Comment:
This test works well, but I think it should go a step further. One of the features with the back button is that the user can go back without having to reload anything. Can you modify this test to do a search, wait for loading spinners to go away, visit the about page, click the back button and then assert that something on the overview tab is exists?

Reply:
Never mind, this one might not be a problem. It turns out that attains is completely down this morning.